Yoast/javascript
# P2 681 add inherit sidebar block

## Summary


This PR can be summarized in the following changelog entry:

* [@yoast/schema-blocks] Adds Inherit-Sidebar instruction.

## Relevant technical choices:

* added an inherit-sidebar instruction which allows subblocks to show the selected parents' sidebars. 
## Test instructions

This PR can be tested by following these steps:

* add `{{ inherit-sidebar }}` to the template that resides in an innerblock of another template; e.g. add this to the `steps` block template of the `yoast/recipe` block template (the premium pr for p2-681 already does this for the job blocks)
* add the template in a new post; select any of the job posting subblocks from the template. Notice the sidebar automatically opens.
* navigate between the innerblock and parent block; notice how the block name (top right) changes to indicate the current block, but the sidebar for the parent block remains.
* add a core/paragraph block to any of the job posting subblocks; notice how the sidebar changes to the core/paragraph sidebar.

## Impact check

* This PR affects the following parts of the plugin, which may require extra testing:
  *

## UI changes

## Quality assurance


Fixes #
